openshift/deploy/deploymentconfig/app.yml

```diff
@@ -70,6 +70,8 @@ objects:
             value: https://ciip-portal-${OC_PROJECT}-feedback.pathfinder.gov.bc.ca
           - name: HOST
             value: https://ciip-portal-${OC_PROJECT}.pathfinder.gov.bc.ca
+          - name: PROJECT
```

Comment:
can you rename the env variable to `NAMESPACE`? "Project" is the openshift term, because they like to make things confusing, but if our app moves to k8s one day then I'd rather use the k8s nomenclature